Python/Product/Debugger/Debugger/DebugEngine/Remote/PythonRemoteEnumListDebug.cs

```diff
@@ -0,0 +1,70 @@
+// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.VisualStudio;
+
+namespace Microsoft.PythonTools.Debugger.Remote {
+    internal class PythonRemoteEnumListDebug<T>
+        where T : class {
```

Comment:
We have one for arrays. But this needs a list so that items can be added and removed.

We could get rid of the array one, and switch all existing uses of that to lists with this code.